1_The_Physical/Guess_Point_Level_1_Stateless/1_Red_Green_Refactor/1_1_Fizzbuzz/src/index.spec.ts

```diff
@@ -1,5 +1,41 @@
+import FizzBuzz from './fizzbuzz';
 
-describe("fizzbuzz", () => {
+describe('FizzBuzz', () => {
+    test('should only accept strings', () => {
```

Comment:
This should be "return strings", but I wonder if this test is redundant since TS will thrown an error if you try to return anything but a number.

Reply:
Good pickup, I will update this test to correct the description as well as covering both string and error scenarios

Reply:
done!